src/AWS.Logger.Core/Core/AWSLoggerCore.cs

```diff
@@ -461,6 +461,22 @@ private async Task<string> LogEventTransmissionSetup(CancellationToken token)
             return currentStreamName;
         }
 
+        /// <summary>
+        /// generate a logstream name
```

Comment:
Please follow the same capitalization pattern found elsewhere

Reply:
Fixed